### PR TITLE
7396 individual page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7396

### CODE REVIEW

File: source/developer.html.erb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+  <div class="section testimonial">
+    <div class="bubble-container">
+      <div class="bubble-section quote">
+        <div class="section-block bubble">
+          <div class="left-quote">
+            &ldquo;
+          </div>
+          <div class="section-block-copy vertical">
+            Lorem ipsum dolor sit amet, consecteture adipiscing elit. Cras rutrum felis
```
Comment (destitutus) [inline source/developer.html.erb]: Is it ready for merge to master?))

File: source/developer.html.erb [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+          </div>
+        </div>
+        <div class="teacher">
+          <div class="profile">
+            <%= image_tag 'assets/developer/duck_duck_logo.png', class: 'profile-img' %>
+          </div>
+          <div class="copy">
+            <h4>DuckDuckGo</h4>
+            <p>
+              bla bla bla<br/>
```
Comment (destitutus) [inline source/developer.html.erb]: Is it ready for merge to master?))

**Comment by MaximKraev** [comment]
@destitutus anything to comment?

**Comment by destitutus** [comment]
> anything to comment?

looks ok now

**Comment by mloginov** [comment]
:+1: ok for me

**Pull request merged**